docs/workflows.rst

```diff
@@ -112,7 +112,7 @@ from additional member(s) before it is accepted as a translation.
 +------------------------+-------------+------------------------------------+
 | Suggestion voting      | off         |                                    |
 +------------------------+-------------+------------------------------------+
-| Autoaccept suggestions | 1           | You can set higher value to        |
+| Auto-accept suggestions| 1           | You can set higher value to        |
```

Comment:
The UI should be changed as well when changing this in the documentation...

Reply:
I added every "autoaccept" I could find to the PR.